generators/generator-bot-assistant-core/README.md

```diff
@@ -1,41 +1,94 @@
 # @microsoft/generator-bot-assistant-core [![NPM version][npm-image]][npm-url]
-> Yeoman generator for creating an Adaptive bot built on the Azure Bot Framework using the Assistant Core template.
 
-## Installation
+> Yeoman generator for creating an assistant-style bot with NLP built on the Azure Bot Framework & Composer.
 
-First, install [Yeoman](http://yeoman.io) and @microsoft/generator-bot-assistant-core using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)):
+The Assistant Core template creates an assistant-style conversational bot containing dialogs for the common conversational flows, with example language models and triggers routing to those dialogs. Includes support for:
+
+- Greeting new and returning users
+- Asking for help
+- Cancelling a dialog
+- Submitting feedback about the bot
+- Error handling in conversations
+- Repeat the previous question
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
+- Disambiguation of NLP results
+
+## What this template is for
```

Comment:
I feel like "Overview" or "Purpose" would be more concise here, especially since we have the "Use this template if you want to..." language right after

Reply:
Yeah, it does get a little long. I kind of feel like altering the packages section to like "Features", moving all of this there, and then calling out if any of those features are packages might work better?